### DISCUSSION TITLE
Custom PropertyGrid crashed app 3.6.0 app

### DISCUSSION THREAD

**Discussion by salah-walid** [iTwin/itwinjs-core#5491] [Q&A] [answered]
We are using PropertyGrid to show selection properties, was working fine in 3.1, in 3.6 it crashes the app.
I see that PropertyGrid is deprecated and should use VirtualizedPropertyGrid, the only problem is that it requires more props, mainly `model` and `eventHandler`.
My question is, is there an example how to use VirtualizedPropertyGrid ?

**Comment by raplemie** [comment, marked as answer]
Hi, can you see if you can use `VirtualizedPropertyGridWithDataProvider` instead of `VirtualizedPropertyGrid`, it should take very similar props than the previous `PropertyGrid`, including the `dataProvider`.

**Reply by salah-walid** [reply]
Thanks, this works, but i noticed it still crashes the app when clicking to show selected item when the iModel has a timeline.
